### DISCUSSION TITLE
COTS UE attach issue: Late: 0; Underflow: 1; Overflow: 0

### DISCUSSION THREAD

**Discussion by mrahman2-vt** [srsran/srsRAN_Project#316] [General Help]
Hi There,
I am planning to connect google pixel to my SA 5G network, phone doesn't connect to 5G network. I am seeing Late: 0; Underflow: 1; Overflow: 0 while running gNB, is this right thing I should see in my terminal?

**Comment by brendan-mcauliffe** [comment]
Hi @mrahman2-vt, seeing an underflow every now and then is fine. As a way to avoid this make sure you have run the performance script found [here](https://github.com/srsran/srsRAN_Project/tree/main/scripts).

Following the [tutorial](https://docs.srsran.com/projects/project/en/latest/tutorials/source/cotsUE/source/index.html#troubleshooting) is the best way to know if you are taking the correct steps to connect your UE. The troubleshooting section outlines some reasons why your phone may not be able to see and/or connect to the network.
